usr/share/gamescope-session-plus/sessions.d/playtron

```diff
@@ -26,8 +26,10 @@ if lspci | grep -i vga | grep -i nvidia > /dev/null; then
 	# required for Nvidia GPUs to work with webview
 	# breaks webview for AMD and Intel GPUs
 	WEBKIT_DISABLE_DMABUF_RENDERER=1
+	GAMESCOPE_DISABLE_EXPLICIT_SYNC=1
 else
 	WEBKIT_DISABLE_DMABUF_RENDERER=0
+	GAMESCOPE_DISABLE_EXPLICIT_SYNC=0
```

Comment:
We don't need this.

Reply:
Isn't `WEBKIT_DISABLE_DMABUF_RENDERER=0` technically also not needed?

Reply:
Probably, but I am not 100% sure. Maybe we can get rid of the else and test it out.

Reply:
I suppose we can leave these both here in case the defaults ever change.